server/src/main/java/io/crate/execution/ddl/tables/CreateTableRequest.java

```diff
@@ -163,7 +158,6 @@ public CreateTableRequest(StreamInput in) throws IOException {
             this.routingColumn = in.readOptionalString();
             this.tableColumnPolicy = ColumnPolicy.VALUES.get(in.readVInt());
             this.partitionedBy = in.readList(StreamInput::readStringList);
-            this.indices = in.readMap();
```

Comment:
This is already in the `onOrAfter(Version.V_5_4_0)` block, so no extra streaming compatiblity necessary